src/models/free-busy.ts

```diff
@@ -4,7 +4,34 @@ import Attributes from './attributes';
 export type FreeBusyQuery = {
   startTime: number;
   endTime: number;
-  emails: string[];
+  emails?: string[];
+  calendars?: FreeBusyCalendarsProperties[];
+};
+
+export type FreeBusyCalendarsProperties = {
+  accountId: string;
+  calendarIds: string[];
+};
+
+export class FreeBusyCalendars extends Model
```

Comment:
```suggestion
export class FreeBusyCalendar extends Model
```

Reply:
Good call out! Commit `04a3593` addresses this.